extension/src/app.js

```diff
@@ -113,23 +113,23 @@ function splashScreen() {
                     "<br/>" +
                     "You have full control of the data collected from your browser. <b>What we study is not you, but how PornHub treats you</b>. Click on the extension icon to access your page." +
                     "<br/>" +
-                    "The collection might happen also in incognito mode. Can be disabled. Also, we find out <a href='/incognito' target=_blank>PornHub might re-link your navigation too</a>!" +
+                    "The collection might also happen in incognito mode (it can be disabled). Also, we find out <a href='/incognito' target=_blank>PornHub might re-link your navigation too</a>!" +
```

Comment:
Unsure how to rewrite "PornHub might re-link your navigation too" because I don't know what it refers to (what does it link to?)

Reply:
* the link "/incognito" do not exist yet
* our finding is the presence of some material in the localstorage which might allow re-linking of activities happened in pornhub incognito / not-incognito, and this is sent to a third party. We should write a technical analysis of the issue it in the link.